### PR TITLE
Expiry calendar holds expiry days variable. Set using app config

### PR DESCRIPTION
**Pull request by phillipDenness**
AAT has it's own application.yaml which must have the same expirydeadline value as the main application.yaml.

### CODE REVIEW

File: src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/DeadlineSteps.java [hunk 1]
```diff
@@ -107,7 +108,7 @@ public void questionsDeadlinesHaveBeenSuccessfullyExtended(String questionExpect
         List<QuestionResponse> questionResponses = allQuestionsResponse.getQuestions();
 
         Calendar expectedExpiryDate = new GregorianCalendar(TimeZone.getTimeZone("GMT"));
-        expectedExpiryDate.add(Calendar.DAY_OF_YEAR, 7 + 7); // deadline for answer + extension
+        expectedExpiryDate.add(Calendar.DAY_OF_YEAR, ExpiryCalendar.deadlineExtensionDays + ExpiryCalendar.deadlineExtensionDays); // deadline for answer + extension
```
Comment (johnhosolirius) [inline src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/DeadlineSteps.java]: ExpiryCalendar.deadlineExtensionDays * 2 would be nicer

File: src/main/java/uk/gov/hmcts/reform/coh/service/utils/ExpiryCalendar.java [hunk 2]
```diff
@@ -1,25 +1,33 @@
...
 
+@Component
 public class ExpiryCalendar {
 
-    private ExpiryCalendar() {}
-    /**
-     * Hard-coded to 7 days.
-     * @return
-     */
+    public static int deadlineExtensionDays;
```
Comment (johnhosolirius) [inline src/main/java/uk/gov/hmcts/reform/coh/service/utils/ExpiryCalendar.java]: Better to have this private otherwise another class could change it

Comment (phillipDenness) [reply src/main/java/uk/gov/hmcts/reform/coh/service/utils/ExpiryCalendar.java]: I'ts not possible to have spring inject on a static variable like you would a local variable. Hence the hacky workaround

Comment (johnhosolirius) [reply src/main/java/uk/gov/hmcts/reform/coh/service/utils/ExpiryCalendar.java]: Okay. Just make it private then

Comment (phillipDenness) [reply src/main/java/uk/gov/hmcts/reform/coh/service/utils/ExpiryCalendar.java]: I've changed the class to use INSTANCE and this should encapsulate the deadline days variable properly.

File: src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/DeadlineSteps.java (hunk 1 above)
Comment (phillipDenness) [reply src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/DeadlineSteps.java]: Done

**Pull request merged**